testcases/OpTestHtxBootme.py

```diff
@@ -0,0 +1,330 @@
+            self.con = self.cv_SYSTEM.cv_HOST.get_ssh_connection()
+            time.sleep(10)
+            for j in range(5):
+                cmd = 'htxcmdline -query  -mdt %s' % self.mdt_file
+                res = self.con.run_command_ignore_fail(cmd, timeout=60)
+                if ("/usr/lpp/htx/mdt/mdt.all") in res[2]:
+                    break
+                else:
+                    time.sleep(10)
+                    log.debug("Mdt start is still in progress")
+            self.con.run_command(cmd)
+            htxerr_file = self.check_remote_file_size_command('wc -c {}'.format("/tmp/htx/htxerr"))
+            if int(htxerr_file.split()[0]) != 0:
+                self.fail("check error logs for exact error and failure")
+            log.info("Reboot cycle %s completed successfully" % (i+1))
+            reboot_time = time.time() - start_time
+            remaining_wait_time = total_wait_time - reboot_time
+            if remaining_wait_time > 0 and i < (self.boot_count-1):
+                log.info("Waiting for next reboot cycle")
+                time.sleep(remaining_wait_time)
+        log.info("Htx Bootme test is completed")
+
+    def is_system_online(self):
+        """
+        This function pings to the host ip and checks system's availability.
+        
+        :return: True if the system is pinging.
+                 False if system is not pinging.
+        """
+        cmd = ["ping", "-c 2", self.cv_HOST.ip]
+        i_try = 3
+        while(i_try != 0):
+            ping = subprocess.Popen(cmd,
+                                    stdin=subprocess.PIPE,
+                                    stdout=subprocess.PIPE,
+                                    universal_newlines=True,
+                                    encoding='utf-8')
+            stdout_value, stderr_value = ping.communicate()
+            if(stdout_value.__contains__("2 received")):
+                return True
+            else:
+                time.sleep(2)
+                i_try -= 1
+        return False
+
+    def check_remote_file_size_command(self, command):
+        """
+        Creates a new SSH client connection, executes a command,
+        and then closes the connection.
+
+        :param command: Command to execute on the remote machine.
+        :return: Command output as a string. 
+                 or Returns Execption error msg if there was an error.
+        """
+        try:
+            client = paramiko.SSHClient()
+            client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+            client.connect(self.cv_HOST.ip, 22, self.cv_HOST.user, self.host_password)
+            stdin, stdout, stderr = client.exec_command(command)
+            output = stdout.read().decode().strip()
+            client.close()
+            return output
+        except Exception as e:
+            print(f"An error occurred: {e}")
+            return e
+
+    def wait_for_reboot_completion(self, ip_addr, timeout=500):
+        """
```

Comment:
we can use existing wait_lpar_state in hmc utils

Reply:
@abdhaleegit The function wait_lpar_state() doesn't return any value to verify. It just raises the exception if the lpar is not in the expected state within the timeout period. The function is useful if just want to print the lpar state.